app/furniture/marketplace/delivery_areas/index.html.erb

```diff
@@ -5,8 +5,9 @@
     <div class="mt-3 grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4">
     <% delivery_areas.each do |area| %>
       <%= render CardComponent.new(classes: "py-2") do %>
-        <%= area.label %>
-        <%= number_to_currency(area.price) %>
+        <%= link_to marketplace.location(:edit, child: :delivery_area) do %>
```

Comment:
Why can't I just use `link_to edit_room_marketplace_delivery_area_path`?

Reply:
In cases where the space is on a custom domain, it would be `edit_room_marketplace_delivery_area_path(room, marketplace, area)` but in cases where the space is on the shared domain it would be `edit_space_room_marketplace_delivery_area_path(space, room, marketplace, area)`

`area.location(:edit)` returns `[:edit, space, room, marketplace, area]`, which Rails' default `polymorphic_url`  implementation would map to `edit_space_room_marketplace_delivery_area(space, room, marketplace, area)`, and our implementation would map to as well, unless the space has a domain, in which case it would map to `edit_room_marketplace_delivery_area`